README.md

```diff
@@ -2,6 +2,20 @@
 
 ![gif](http://g.recordit.co/zjYe5COyVB.gif)
 
-This Electron application sets your Mac OS desktop background with images from Unsplash. The app is built with a Vue.js framework that implements localStorage, axios requests, and AppleScript.
+This app lets you quickly and easily change your desktop background image to something much cooler than whatever it currently is.
 
-Boilerplate from [electron-vue](https://github.com/SimulatedGREG/electron-vue) using [vue-cli](https://github.com/vuejs/vue-cli). Documentation about this boilerplate can be found [here](https://simulatedgreg.gitbooks.io/electron-vue/content/index.html).
+### Technologies and resources used:
+
+[Electron](http://electron.atom.io) - A framework for building cross-platform applications with web technologies. Electron is pretty sweet.
```

Comment:
I like how you broke out these technologies and added more descriptions to each! 👍